app/models/user.rb

```diff
@@ -0,0 +1,31 @@
+class User < ApplicationRecord
+  has_many :tasks, dependent: :destroy
+
+  validates :name, presence: true, length: { maximum: 30 }
+  validates :email, presence: true, length: { maximum: 255 },
+                  format: { with: /\A[\w+\-.]+@[a-z\d\-.]+\.[a-z]+\z/i },
+                  uniqueness: true
+  before_validation { email.downcase! }
+  validates :password, presence: true, length: { minimum: 6 }
+
+  has_secure_password
+
+  enum admin: { user: false, admin: true }
+
+  before_destroy :do_not_destroy_last_one_admin
+  before_update :do_not_change_last_one_admin
+
+  private
```

Comment:
privateの後は1行開けるようにしましょう。
```suggestion
  private
  
```

Reply:
9f0c135 でフォーマットの修正を確認